pkg/session/bootstrap.go

```diff
@@ -3267,6 +3272,15 @@ func upgradeToVer216(s sessiontypes.Session, ver int64) {
 	mustExecute(s, "UPDATE mysql.global_variables SET VARIABLE_VALUE='table' WHERE VARIABLE_NAME = 'tidb_scatter_region' AND VARIABLE_VALUE = 'ON'")
 }
 
+func upgradeToVer217(s sessiontypes.Session, ver int64) {
+	if ver >= version217 {
+		return
+	}
+	// If tidb_schema_cache_size does not exist, insert a record and set the value to 0
+	// Otherwise do nothing.
+	mustExecute(s, "INSERT IGNORE INTO mysql.global_variables VALUES ('tidb_schema_cache_size', 0)")
```

Comment:
if there's no such row in `global_variables`

when we execute this statement, the first domain and infocache has already started, the infocache is still v2 ?

Reply:
Later when infoschema reload it would detect the new infoschema value and change to v1  @D3Hunter